recipes/archive/justfile

```diff
@@ -21,6 +24,8 @@ tree     := join('archive', 'pc', 'mod')
     just pack 'Addicted.VFX'
     just pack 'Addicted.Biomon'
     just pack 'Addicted.Translations'
+    just pack 'Addicted.DrugPump'
+    just xl 'DrugPump'

```

Comment:
same as above: since you follow the same naming convention, you don't need this anymore,
please revert it .
of course, you need to keep `just pack 'Addicted.DrugPump'`